src/pybind/mgr/dashboard/servers.html

```diff
@@ -24,9 +24,16 @@
             };
 
             rivets.formatters.short_version = function(version) {
-                // Turn "ceph version 1.2.3-g9asdasd (as98d7a0s8d7)"
-                // into "1.2.3-g9asdasd"
-                return /ceph version (.+) \(.+\)$/.exec(version)[1];
+                // Expect "ceph version 1.2.3-g9asdasd (as98d7a0s8d7)"
+                result = /ceph version (.+) \(.+\)/.exec(version);
```

Comment:
can we make a structured ceph version thing?  i was considering changing the format of this string a couple weeks ago and didn't out of fear of things like this.  but we have 2 new fields added at the end now (release name, e.g. 'luminous') and release type ('dev', 'rc', 'stable').